ci/worker_files/worker-config

```diff
@@ -0,0 +1,8 @@
+WORKER_MODE="0"
+PKT_IFACE="p2p1"
```

Comment:
Wait why do you have a different iface for pktgen and for mtcp? I'm pretty sure they can just share one

Reply:
I had a hard time configuring the two interfaces separately. When one runs, it's ok, but configuring two interfaces is much easier than checking to see if the server was lying, before each check.

Reply:
For now maybe we let it slide, but I'm not convinced that this is better. What's the topology you're using(which node is connected where), 2 NICs instead of one doesn't make much sense as it's using more resources

Reply:
This is a tricky one, on the manager side (nimbnode17 for example) we only need 1 NIC, true, but on the client, we don't have to do anything if we can send Pktgen as one IP, and mTCP as another. This allows us to basically not touch the client server, because it's already configured. Your choice, but this makes it easy to we can run just mTCP, or just Pktgen, without any issues. This makes it faster, but I can definitely work a solution if necessary.